overlord/hookstate/ctlcmd/set.go

```diff
@@ -38,6 +38,9 @@ type setCommand struct {
 		PlugOrSlotSpec string   `positional-arg-name:":<plug|slot>"`
 		ConfValues     []string `positional-arg-name:"key=value"`
 	} `positional-args:"yes"`
+
+	String bool `short:"s" description:"parse as a string"`
+	Typed  bool `short:"t" description:"parse strictly as JSON document"`
```

Comment:
minor: I'd be more explicit and replace "parse" with "parse the value".

Was the choice of the letter `t` suggested to you by some other team member? Just wondering, because a `-j` option for JSON would have sounded more obvious to me.

Reply:
Thanks @mardy! I'll improve the description, as per your suggestion. Regarding the choice of letter: it was, yes, I think because `-t` is already an option for `snap get`